sled-agent/src/params.rs

```diff
@@ -476,22 +479,23 @@ impl ServiceZoneRequest {
     }
 
     // XXX: any reason this can't just be service.to_string()?
-    pub fn srv(&self, service: &ServiceType) -> SRV {
+    pub fn srv(&self, service: &ServiceType) -> Option<SRV> {
```

Comment:
The switch zone setup script is just a oneshot script so it shouldn't have a SRV attached to it, I think. Hence the `Option<SRV>`.